editoast/src/views/v2/train_schedule.rs

```diff
@@ -486,10 +471,10 @@ fn train_simulation_input_hash(
     format!("simulation_{osrd_version}.{infra_id}.{infra_version}.{hash_simulation_input}")
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, IntoParams)]
-struct SimulationBatchParams {
+#[derive(Debug, Clone, Deserialize, ToSchema)]
+struct SimulationBatchForm {
     infra: i64,
```

Comment:
I believe we try to have always `infra_id` for naming an identifier (else, we don't know if it's an `Infra`).
```suggestion
    infra_id: i64,
```